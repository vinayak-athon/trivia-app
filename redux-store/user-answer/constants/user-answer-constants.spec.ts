import { UserAnswerActionConsts } from ".";

describe("UserAnswerActionConsts", () => {
  it("should have to Match Snapshot", () => {
    expect(UserAnswerActionConsts).toMatchSnapshot();
  });
});
