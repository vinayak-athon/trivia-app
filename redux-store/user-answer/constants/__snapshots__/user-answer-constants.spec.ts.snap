// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserAnswerActionConsts should have to Match Snapshot 1`] = `
Object {
  "RESET_USER_ANSWER": "RESET_USER_ANSWER",
  "SET_USER_ANSWER": "SET_USER_ANSWER",
}
`;
